team/index.html

```diff
@@ -1429,61 +1429,7 @@ <h1>Developers</h1>
 						<span>Android Developer</span>
 					</div>
 				</div>
-
-				<div class="col-md-3 col-sm-6 speaker-column">
-					<div class="speaker">
-						<div class="image-holder">
-							<img class="background-image" alt="Abhinav Raj" src="../img/AbhinavRaj.jpg">
-							<div class="hover-state text-center preserve3d">
-								<div class="social-links vertical-align">
-									<a href="https://github.com/abhinavraj23"><i class="icon fa fa-github"></i></a>
-									<a href="https://www.facebook.com/abhinav.raj.583"><i class="icon social_facebook"></i></a>
-									<a href="https://www.linkedin.com/in/abhinav-raj-234497159/"><i class="icon social_linkedin"></i></a>
-									<a href="https://twitter.com/AbhinavApril"><i class="icon social_twitter"></i></a>
-								</div>
-							</div>
-						</div>
-						<span class="speaker-name">Abhinav Raj</span>
-						<span>Android Developer</span>
-					</div>
-				</div>
-
-				<div class="col-md-3 col-sm-6 speaker-column">
-					<div class="speaker">
-						<div class="image-holder">
-							<img class="background-image" alt="Bhavesh Anand" src="../img/bhaveshAn.png">
-							<div class="hover-state text-center preserve3d">
-								<div class="social-links vertical-align">
-									<a href="https://github.com/bhaveshAn"><i class="icon fa fa-github"></i></a>
-									<a href="https://www.linkedin.com/in/bhaveshan/"><i class="icon social_linkedin"></i></a>
-									<a href="https://twitter.com/bhaveshAn_"><i class="icon social_twitter"></i></a>
-									<a href="http://bhaveshan.github.io"><i class="icon fa fa-globe"></i></a>
-								</div>
-							</div>
-						</div>
-						<span class="speaker-name">Bhavesh Anand</span>
-						<span>Developer</span>
-					</div>
-				</div>
-
-				<div class="col-md-3 col-sm-6 speaker-column">
-					<div class="speaker">
-						<div class="image-holder">
-							<img class="background-image" alt="Praveen Ojha" src="../img/praveen_ojha.jpg">
-							<div class="hover-state text-center preserve3d">
-								<div class="social-links vertical-align">
-									<a href="https://github.com/praveenojha33"><i class="icon fa fa-github"></i></a>
-									<a href="https://www.facebook.com/praveen.ojha.3511"><i class="icon social_facebook"></i></a>
-									<a href="https://www.linkedin.com/in/praveen-ojha-59521669/"><i class="icon social_linkedin"></i></a>
-									<a href="https://twitter.com/PraveenOjha5"><i class="icon social_twitter"></i></a>
-									<a href="http://praveenojha.me"><i class="icon fa fa-globe"></i></a>
-								</div>
-							</div>
-						</div>
-						<span class="speaker-name">Praveen Ojha</span>
-						<span>Developer</span>
-					</div>
-				</div>
+				<div class="clearfix"></div>
```

Comment:
What is the use of this line?

Reply:
I just checked now nothing got affected after removing it, so can I remove it @samagragupta

Reply:
Actually this line helps when we add code after that block if we will not add this then there will be some vacant space that's why I didn't remove it @samagragupta